src/components/elements/Inputs/FileInput/RHFFileInput.tsx

```diff
@@ -232,12 +243,20 @@ const RHFFileInput = ({
 
   return (
     <FileInput
-      {...fileInputProps}
       files={files}
+      {...(isPhotosAndVideo && {
+        previewAsTable: true,
+        descriptionInput: "drag and drop or browse your device",
```

Comment:
Checking if this component is going to be translated

Reply:
Done.